records_mover/records/sources/fileobjs.py

```diff
@@ -154,14 +155,28 @@ def to_dataframes_source(self,
             else:
                 chunksize = int(entries_per_chunk / num_fields)
 
+            def fix_integer_columns(dfs: Iterator['Dataframe']) -> Iterator['Dataframe']:
+                # TODO: Move generator expression using that function
+                for df in dfs:
+                    if 'convert_dtypes' in dir(df):
+                        # Allow nullable integers to be represented
+                        df = df.convert_dtypes(convert_integer=True)
+                    else:
+                        logger.warning("Using old version of pandas; "
+                                       "not able to represent nullable integer columns")
+                    yield df
+
+            # TODO: switch to generator expressions: https://treyhunner.com/2018/06/how-to-make-an-iterator-in-python/
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker